### ISSUE TITLE
Restoring some old files

### ISSUE THREAD

**Issue by Storm3000** [elishacloud/Silent-Hill-2-Enhancements#567]
Hi, everyone.

Congratulations for the AMAZING and WONDERFUL job that you, guys, have done. I never imagined this game could be fixed that much.

As much as I love what you've done with the Enhanced Edition, I'm a little more 'old school' gamer and I prefer to play with the original pics, but preserving all the fixes you've added. I remember that @Polymega or someone else made some time ago a beautiful title screen in HD based on the PS2 version. Sadly, I lost that file a while ago.

Could you please share that file?

**Comment by Polymega** [comment]
The PS2 main menu is found by downloading the Essential Files pack then toggling this feature on:

![image](https://user-images.githubusercontent.com/27395260/179381302-6339a5f0-9830-4766-8e7a-fdcd79206dd5.png)

**Comment by Storm3000** [comment]
@Polymega Thank you very much. But I was talking about this one: https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/48#issuecomment-464410040

Or some similar model that had parts of creatures or something like that inside the letters. I can't find it anymore. The one included here in the EE is in low res.

**Comment by Polymega** [comment]
Hmm, the only high res version of the main menu we shipped out is the SH2:EE main menu. That menu uses solid black for the logo. The link you shared was a WIP I made early on.

I believe your description of creatures being inside the letters is actually the "wave" from the background continuing on behind the letters, which is seen in the North American [PS2](https://files.hiddenpalace.org/thumb/c/c7/Capture.PNG/1200px-Capture.PNG) and [Xbox](https://gameir.ie/wp-content/uploads/f5a8d818fb64e9f0b87bb9023299e310-750x400.jpg) main menus. 

We've included the North American PS2 main menu in the Essential Files pack, which can be seen by toggling the "disable 'sh2e\pic\ path" option in the Config Tool, for those who want to play using the original pics like your request.
